src/tests/ftest/util/daos_core_base.py

```diff
@@ -179,6 +173,33 @@ def run_subtest(self):
                 self.fail(
                     "{0} failed with return code={1}.\n".format(
                         job_str, result.result.exit_status))
+        finally:
+            # List any remote cmocka files
+            self.log.debug("Remote %s directories:", cmocka_dir)
+            ls_command = "ls -al {}".format(cmocka_dir)
+            log_task(" ".join([get_clush_command(self.hostlist_clients, "-B -S"), ls_command]))
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'command' in function call